Doc/library/multiprocessing.rst

```diff
@@ -1168,6 +1168,61 @@ For example:
     the data in the pipe is likely to become corrupted, because it may become
     impossible to be sure where the message boundaries lie.
 
+Custom Reduction
+~~~~~~~~~~~~~~~~
+
+.. currentmodule:: multiprocessing
+
+Several primitives of the :mod:`multiprocessing` module such as
+:class:`multiprocessing.Queue`, :class:`multiprocessing.connection.Listener` or
+:class:`multiprocessing.connection.Server` need to serialize and deserialize Python
+objects to communicate between processes. Sometimes is useful to control what
+serialization is to be used for the transport of data for supporting communication with
+different versions of Python, use more performant 3rd party tools or custom strategies.
+
+For this pourpose a set of hooks is available to provide alternate implementations of
+the reduction mechanism:
+
+.. currentmodule:: multiprocessing.reduction
+
+.. class:: AbstractReducer()
+
+   Abstract base class for use in implementing a Reduction class suitable for use
+   in replacing the standard reduction mechanism used in multiprocessing.
+
+   .. function:: get_pickler_class():
+
+      This method must return a subclass of :class:`pickle.Pickler` to be used by
+      the multiprocessing reducer mechanism.
+
+.. currentmodule:: multiprocessing
+
+.. function:: set_reducer(reduction)
+
+   Sets a reduction class to be used for serialization and deserialization by the module
+   primitive internals. **reduction** must be a subclass of
```

Comment:
Hmm, by reading the code it seems `reduction` should be an _instance_ of `AbstractReducer`, not a subclass. Either way, docs and code should agree with each other.